integration_tests/src/main/python/hash_aggregate_test.py

```diff
@@ -1088,3 +1089,44 @@ def do_it(spark):
         df = two_col_df(spark, StringGen('k{1,5}'), ArrayGen(MapGen(StringGen('a{1,5}', nullable=False), StringGen('[ab]{1,5}'))))
         return df.groupBy('a').agg(f.min(df.b[1]["a"]))
     assert_gpu_and_cpu_are_equal_collect(do_it)
+
+
+# Tests for standard deviation and variance aggregations.
```

Comment:
we should add the Stddev* and Variance* aggs to the ANSI fallback tests here (and need make sure they do fallback).

Reply:
No we don't because all the only type is supports is double. So it will just work.

Reply:
> No we don't because all the only type is supports is double. So it will just work.

I stand corrected, everything is doubles with this agg!